apps/docs/components/collapsible/Collapsible.tsx

```diff
@@ -1,54 +1,31 @@
 "use client";
 
 import clsx from "clsx";
-import { type ReactNode, useEffect, useRef, useState } from "react";
-import { ToggleButton } from "react-aria-components";
+import type { ReactNode } from "react";
+import { Button, UNSTABLE_Disclosure as Disclosure, UNSTABLE_DisclosurePanel as DisclosurePanel } from "react-aria-components";
 import { Icon, CollapseIcon } from "@/components/icon";
 
 import "./collapsible.css";
 
 export interface CollapsibleProps {
     children: ReactNode;
     title: ReactNode;
-    label?: string;
-    isOpen?: boolean;
```

Comment:
why is this removed? we never want to decide if it's open by default? In that case it would probably be called defaultOpen.  Or the RAC uses defaultExpanded.

Reply:
I removed them since isOpen was used to open / close the custom disclosure, now that we use RAC component, this is offloaded to them. If you see a use for this I can add it. Since it's for the documentation and there's no current use cases I'm not sure it's worth.